repository/s.json

```diff
@@ -2836,10 +2836,20 @@
 		{
 			"name": "SortBy",
 			"details": "https://github.com/Doi9t/SortBy",
+			"description": "A Sublime Text plugin that allows you to sort lines with methods that are not present by default",
```

Comment:
Description is automatically fetched from the repo so I think it's preferred to not add it here.

Reply:
I removed the description; thanks!